packages/gasket-plugin-analyze/lib/get-commands.js

```diff
@@ -2,19 +2,16 @@
  * Get the analyze command
  *
  * @param {Gasket} gasket - Gasket
- * @param {Object} GasketCommand - Base Gasket command to extend
  * @returns {GasketCommand} command
  */
-module.exports = function getCommands(gasket, { GasketCommand }) {
-
-  class AnalyzeCommand extends GasketCommand {
-    async gasketRun() {
-      await this.gasket.exec('build');
+module.exports = function getCommands(gasket) {
+  const AnalyzeCommand = {
```

Comment:
nit: PascalCase doesn't make sense here now, but what about just inlining this? 
```suggestion
  return {
```